mipidsi/src/models/gc9107.rs

```diff
@@ -0,0 +1,101 @@
+use display_interface::{DataFormat, WriteOnlyDataCommand};
+use embedded_graphics_core::{pixelcolor::Rgb565, prelude::IntoStorage};
+use embedded_hal::delay::DelayNs;
+
+use crate::{
+    dcs::{
+        BitsPerPixel, ExitSleepMode, PixelFormat, SetAddressMode, SetDisplayOn, SetInvertMode,
+        SetPixelFormat, WriteMemoryStart,
+    },
+    error::Error,
+    options::ModelOptions,
+};
+
+use super::{Dcs, Model};
+
+/// GC9A01 display in Rgb565 color mode.
+pub struct GC9107;
+
+impl Model for GC9107 {
+    type ColorFormat = Rgb565;
+    const FRAMEBUFFER_SIZE: (u16, u16) = (240, 240);
+
+    fn init<DELAY, DI>(
+        &mut self,
+        dcs: &mut Dcs<DI>,
+        delay: &mut DELAY,
+        options: &ModelOptions,
+    ) -> Result<SetAddressMode, Error>
+    where
+        DELAY: DelayNs,
+        DI: WriteOnlyDataCommand,
+    {
+        let madctl = SetAddressMode::from(options);
+
+        delay.delay_us(200_000);
+
+        dcs.write_raw(0x36, &[0xC8])?;
+
+        dcs.write_raw(0xB0, &[0xC0])?;
+        dcs.write_raw(0xB2, &[0x2F])?;
+        dcs.write_raw(0xB3, &[0x03])?;
+        dcs.write_raw(0xB6, &[0x19])?;
+        dcs.write_raw(0xB7, &[0x01])?;
+
+        dcs.write_raw(0xAC, &[0xCB])?;
+        dcs.write_raw(0xAB, &[0x0E])?;
+
+        dcs.write_raw(0xB4, &[0x04])?;
+
+        dcs.write_raw(0xA8, &[0x19])?;
+
+        dcs.write_raw(0x3A, &[0x05])?;
+
+        dcs.write_raw(0xB8, &[0x08])?;
+
+        dcs.write_raw(0xE8, &[0x24])?;
+
+        dcs.write_raw(0xE9, &[0x48])?;
+
+        dcs.write_raw(0xEA, &[0x22])?;
+
+        dcs.write_raw(0xC6, &[0x30])?;
+        dcs.write_raw(0xC7, &[0x18])?;
+
+        dcs.write_raw(
+            0xF0,
+            &[
+                0x1F, 0x28, 0x04, 0x3E, 0x2A, 0x2E, 0x20, 0x00, 0x0C, 0x06, 0x00, 0x1C, 0x1F, 0x0f,
+            ],
+        )?;
+
+        dcs.write_raw(
+            0xF1,
+            &[
+                0x00, 0x2D, 0x2F, 0x3C, 0x6F, 0x1C, 0x0B, 0x00, 0x00, 0x00, 0x07, 0x0D, 0x11, 0x0f,
+            ],
+        )?;
+
+        dcs.write_raw(0x21, &[])?; // invert display
```

Comment:
Display inversion shouldn't be on by default. Use this instead: https://github.com/almindor/mipidsi/blob/e808fb91758af54fc746d09c092a52c8cb65a25d/mipidsi/src/models/gc9a01.rs#L117